final_assignment/cristina_ungureanu/parser.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# *-* coding: UTF-8 *-*
+
+import os
+import json
+import urllib2
+import subprocess as sp
+import time
+from pprint import pprint
+
+with open('document') as data_file:    
+    data = json.load(data_file)
+
+
+log_file = open("logfile","w")
+log_file.close()
+
+log_file = open("build.log","a")
+
+#before_install
+url_download = data["before_install"][0]["download"]["source"]
+de_descarcat = urllib2.urlopen(url_download)
+fisier_descarcat = open("/home/random/script.sh","w")
+fisier_descarcat.write(de_descarcat.read())
+fisier_descarcat.close()
+log_file.write("Am terminat before_install.\n")
+
+
+log_file.write("Trec la install.\n")
+
+
+#install
+attempts = int(data['install'][0]['run_script']['attempts'])
+check_exit_code = data['install'][0]['run_script']['check_exit_code']
+command = data['install'][0]['run_script']['command']
+cwd = data['install'][0]['run_script']['cwd']
+env_dict = dict(data['install'][0]['run_script']['env_variables'])
+retry_interval = int(data['install'][0]['run_script']['retry_interval'])
+shell = data['install'][0]['run_script']['shell']
```

Comment:
Ca si la condensarea config-urilor.. condensarea in ceva mai readable e o idee foarte buna, dar ar fi putut fi numite mai sugestiv, pe faze ale instalarii:
 - inst_attempts
 - inst_check
 - inst_command, etc.